src/app.ts

```diff
@@ -112,6 +112,13 @@ if (process.env.NODE_ENV === 'production') {
     }
   )
   server.listen(443)
+
+  const httpApp = express()
+  httpApp.get('/*', (req, res) => {
+    const params = req.query ? `?${req.headers.host.split('?')[1]}` : ''
```

Comment:
if for whatever reason.... there is a `?` in the query params, this might fail. I think this is better:
```
req.headers.host.split('?').shift().join('?')
``

Reply:
never mind.